core/templates/dev/head/pages/exploration_editor/editor_tab/StateResponses.js

```diff
@@ -564,16 +566,18 @@ oppia.controller('StateResponses', [
           'explorationContextService', 'editorContextService',
           'explorationStatesService', 'trainingDataService',
           'AnswerClassificationService', 'focusService',
-          'RULE_TYPE_CLASSIFIER',
+          'RULE_TYPE_CLASSIFIER', 'oppiaPlayerService',
```

Comment:
Sorry, @anmolshkl -- I don't understand your question.

That said, I feel it's inappropriate for general services in the editor to depend on oppiaPlayerService. The version of the exploration is available in the editor via other means, I think. @pranavsid98 -- could you please try and figure out how the history tab knows what the current version is?